CONTRIBUTING.md

```diff
@@ -1,23 +1,76 @@
-## How to Paritcipate?
+## How to Contribute?
```

Comment:
Also, Add How to How to Participate section. It is important as many students don't know about it yet.

Reply:
Ok  👍.

Reply:
@rishabh-live I have added "How to Participate"